### DISCUSSION TITLE
higress官方文档完善

### DISCUSSION THREAD

**Discussion by Everything1sPossible** [alibaba/higress#469] [General]
期望贵方可以进一步完善官方文档，比如自定义插件开发，可以多一些API说明等。

**Comment by johnlanni** [comment]
https://github.com/alibaba/higress/issues/415

sorry，实在是精力有限，想通过这个 learing to share 的题目，来召集社区小伙伴贡献这块的文档。

我这边可以提供指导，有兴趣可以联系我
